Makefile

```diff
@@ -4,4 +4,7 @@ clean:
 
 .PHONY: profile
 profile:
-	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets bower_components css index.html js manifest.webapp
+	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets index.html lib node_modules js manifest.webapp
+
+addon:
+	zip addon.zip addon.js manifest.json assets index.html lib node_modules
```

Comment:
Not equipped to test it right now but happy to remove it if not needed :)
Was MozFest before or after the change of format?